### DISCUSSION TITLE
Welcome to projeto-integrado-2SEM-2022 Discussions!

### DISCUSSION THREAD

**Discussion by fernandoleonid** [fernandoleonid/projeto-integrado-2SEM-2022#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by srmt03** [comment]
https://www.figma.com/file/hX5Rv8g9IL5lFkW4FxFwqv/pizzaria?node-id=0%3A1&t=2Mn5NoWKdfV52dT9-0

**Comment by LusCamilo** [comment]
Link pro README do projeto: https://github.com/LusCamilo/README-ProjetoPizzaria
